src/compute-shortname.js

```diff
@@ -142,42 +142,45 @@ function completeWithSeriesAndLevel(shortname, url) {
     }
   }
 
+  const seriesBasename = forkOf ?? shortname;
+  const specShortname = forkOf ? `${forkOf}-fork-${shortname}` : shortname;
```

Comment:
it took me a while to walk the path that led to the conclusion that for the case at hand, `shortname` would have been calculated as `exception-handling` (which is great); I'm not sure where, but a comment that would make that easier to grasp would be helpful.

Reply:
(all the more so that the fact that it worked for webassembly that uses separate repos for its forks may not remain universally true for this mode of composition)

Reply:
I completed the README and CONTRIBUTING files to explain the prefixing logic.